pkg/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -4,4 +4,28 @@ one = "kafka"
 
 [kafka.cmd.shortDescription]
 description = "Short description for command"
-one = "Create, view, use, and manage your Apache Kafka instances"
\ No newline at end of file
+one = "Create, view, use, and manage your Apache Kafka instances"
+
+[kafka.cmd.longDescription]
+description = "Long description for command"
+one = '''
+A collection of commands for managing and interacting with Apache Kafka instances.
+
+You may use `kafka [create/delete/list/describe]` to manage your Kafka instances, `kafka use` to control your currently utilised Kafka instance, and `kafka [topics/consumergroup]` to interact with Kafka instances.
+'''
+
+[kafka.cmd.example]
```

Comment:
For each of these examples, you can probably remove "Apache". Elsewhere, we just refer to "Kafka instances", not "Apache Kafka instances".